db/migrate/20240516101409_updateresourcepoolidentifierstomiqproductfeatures.rb

```diff
@@ -0,0 +1,63 @@
+class Updateresourcepoolidentifierstomiqproductfeatures < ActiveRecord::Migration[6.1]
+  class MiqProductFeature < ActiveRecord::Base; end
+
+  FEATURE_MAPPING_UPDATE = {
+    'resource_pool'           => 'resource_pool_cloud',
+    'resource_pool_view'      => 'resource_pool_cloud_view',
+    'resource_pool_show_list' => 'resource_pool_cloud_show_list',
+    'resource_pool_show'      => 'resource_pool_cloud_show',
+    'resource_pool_control'   => 'resource_pool_cloud_control',
+    'resource_pool_tag'       => 'resource_pool_cloud_tag',
+    'resource_pool_protect'   => 'resource_pool_cloud_protect',
+    'resource_pool_admin'     => 'resource_pool_cloud_admin',
+    'resource_pool_delete'    => 'resource_pool_cloud_delete'
+  }.freeze
+
+  FEATURE_MAPPING_CREATE = {
+    "resource_pool_infra"           => {:name => "Infrastructure Resource Pools", :description => "Access Everything under Infrastructure Resource Pools", :feature_type => "node"},
+    "resource_pool_infra_view"      => {:name => "View", :description => "Display Lists of Infrastructure Resource Pools", :feature_type => "view", :parent => "resource_pool_infra"},
+    "resource_pool_infra_show_list" => {:name => "List", :description => "Display Lists of Infrastructure Resource Pools", :feature_type => "view", :parent => "resource_pool_infra_view"},
+    "resource_pool_infra_show"      => {:name => "Show", :description => "Display Individual Infrastructure Resource Pools", :feature_type => "view", :parent => "resource_pool_infra_view"},
+    "resource_pool_infra_control"   => {:name => "Operate", :description => "Perform Operations on Infrastructure Resource Pools", :feature_type => "control", :parent => "resource_pool_infra"},
+    "resource_pool_infra_tag"       => {:name => "Edit Tags", :description => "Edit Tags of Infrastructure Resource Pools", :feature_type => "control", :parent => "resource_pool_infra_control"},
+    "resource_pool_infra_protect"   => {:name => "Manage Policies", :description => "Manage Policies of Infrastructure Resource Pools", :feature_type => "control", :parent => "resource_pool_infra_control"},
+    "resource_pool_infra_admin"     => {:name => "Modify", :description => "Modify Infrastructure Resource Pools", :feature_type => "admin", :parent => "resource_pool_infra"},
+    "resource_pool_infra_delete"    => {:name => "Remove", :description => "Remove Infrastructure Resource Pools", :feature_type => "admin", :parent => "resource_pool_infra_admin"}
+  }.freeze
```

Comment:
@Guddetisandeep you don't need to create the new features in this PR, that will be done when we seed the database.  The only reason we need schema migration for existing product features is because users might have used the existing records for user/group permissions already and we can't just delete them.